src/app/fyle/add-edit-mileage/add-edit-mileage.page.ts

```
@@ -2415,6 +2415,9 @@ export class AddEditMileagePage implements OnInit {
         const formValue = this.getFormValues();
         let customProperties = res.customProperties;
         customProperties = customProperties?.map((customProperty) => {
+          if (!customProperty.value) {
+            customProperty = this.customFieldsService.setDefaultValue(customProperty, customProperty.type);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)